docs/content/en/docs/contribution-guidelines/piped-development.md

```diff
@@ -25,3 +25,11 @@ bazelisk run --run_under="cd $PWD && " //cmd/piped:piped -- piped \
 --tools-dir=/tmp/piped-bin \
 --config-file=piped-config.yaml
 ```
+
+## How to load piped image into the local Docker client
+
```

Comment:
Thank you so much. Don't we need to mention the need to `make build-images` in advance?

Reply:
No, we don't need it. Tt automatically re-builds the images if any changes are detected.

Reply:
I don't even know that. Cool